### ISSUE TITLE
[Story] Adding For Profit Ownership Name - Back-end

### ISSUE THREAD

**Issue by MitchLivingstone** [department-of-veterans-affairs/va.gov-team#43560]
This story is to add For Profit Ownership Names to institutions cards to provide the end user further school information.

The scope of this story is to update the Back-end.

Acceptance Criteria
- From the attached xls, import Institutions Name into GIDS 

XLS file: https://app.zenhub.com/files/133843125/708d0f5b-ab82-42ad-bde9-384f45524d7e/download

**Comment by PJart** [comment]
Added in ability to download csv and steps to add it to the institution and payload, added some tests, noticed some security vulnerabilities with some gems in the gemfile. Before I can get this merged in and moved over to staging to test, I need to get the gemfile changes to compile and get the ci to pass.

**Comment by PJart** [comment]
Working on getting gemfile to compile, rake ci showed some gems that needed to be updated, said rails update. In process of getting that working so I can move this to staging to test.

**Comment by SebVortex** [comment]
This story should be closed. No more work to be done here.

**Comment by MitchLivingstone** [comment]
Waiting to hear back from the Backend group to be merged into GIDs

**Comment by dsrr47** [comment]
Waiting on a code approval for a backend PR

**Comment by jfinnern** [comment]
Validated in staging
Test run: https://dsvavsp.testrail.io/index.php?/runs/view/3543&group_by=cases:section_id&group_order=asc
Test case: [https://app.zenhub.com/files/133843125/9e7b3c6b-c1f5-43ae-8c09-35e03ba5ff2e/download](https://app.zenhub.com/files/133843125/9e7b3c6b-c1f5-43ae-8c09-35e03ba5ff2e/download)

**Comment by MitchLivingstone** [comment]
Closed and completed.
